src/utils/StargazerLoader.js

```diff
@@ -1,18 +1,11 @@
 import gitHubUtils from "./GitHubUtils";
 import stargazerStats from "./StargazerStats";
 
-const colors = [
-  "#008FFB",
-  "#00E396",
-  "#FEB019",
-  "#FF4560",
-  "#775DD0",
-  "#F86624",
-  "#00B1F2",
-  "#5A2A27",
-];
-
-export const maxReposAllowed = 8;
+// Refer to post https://stackoverflow.com/questions/10014271/generate-random-color-distinguishable-to-humans
+function makeColor(number) {
+  const hue = number * 137.508; // use golden angle approximation
+  return `hsl(${hue},50%,75%)`;
```

Comment:
Let's change the 3rd param to 50% (`hsl(${hue},50%,50%)`)? The colors look better there:

![image](https://user-images.githubusercontent.com/9059541/209452898-341066d6-f5d7-467e-98d7-54eac9290847.png)